### ISSUE TITLE
Error decoding base64 image with charset.

### ISSUE THREAD

**Issue by matatonic** [QwenLM/Qwen2-VL#202]
The error:
```
  File "/app/backend/qwen2-vl.py", line 52, in stream_chat_with_images
    image_inputs, video_inputs = process_vision_info(messages)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/qwen_vl_utils/vision_process.py", line 217, in process_vision_info
    image_inputs.append(fetch_image(vision_info))
                        ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/qwen_vl_utils/vision_process.py", line 92, in fetch_image
    raise ValueError(f"Unrecognized image input, support local path, http url, base64 and PIL.Image, got {image}")
ValueError: Unrecognized image input, support local path, http url, base64 and PIL.Image, got data:image/jpeg;charset=utf-8;base64,/9j/4AAQSkZJRgABAQEAtAC0...........
                                                                                                              ^^^^^^^^^^^^^
---------------------------------------------------------------------------------------------------------------------------
```

From here: 

https://github.com/QwenLM/Qwen2-VL/blob/137465a16b1eac19d26459afafbbcac80d68b371/qwen-vl-utils/src/qwen_vl_utils/vision_process.py#L84-L89

**Comment by matatonic** [comment]
The images were made with this: https://pypi.org/project/python-datauri/

**Comment by kq-chen** [comment]
the bug has been fix by #215 , so I will close this issue.
